recipes/recipes_emscripten/harfbuzz/recipe.yaml

```diff
@@ -11,31 +11,35 @@ source:
   sha256: c758fdce8587641b00403ee0df2cd5d30cbea7803d43c65fddd76224f7b49b88
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
+  # NOTE: harfbuzz doesn't need host dependencies
```

Comment:
We should be using the headers from the `host` platform and not the ones from `build`, in case these have been patched (for example in `glib`). These dependencies should also be listed under host to ensure they are available for the correct platform. `glib` has not be repackaged with the new emscripten.